lib/universal-search.js

```diff
@@ -88,22 +88,47 @@ Search.prototype = {
   _loadStyleSheets: function(win) {
     const doc = win.document;
     const docEl = doc.documentElement;
+    //Check whether platform type is 'Darwin', 'Linux', or 'WINNT'.
+    const osString = Services.appinfo.OS;
```

Comment:
Let's get rid of lines 91-92 and just check `Services.appinfo.OS == foo` directly in the conditionals below